azerothcore/azerothcore-wotlk
# feat(DB): Change VerifiedBuild datatype to MEDIUMINT

## Changes Proposed:
-  Changes `VerifiedBuild` column datatype to `MEDIUMINT` for every applicable table to allow for higher build numbers.

## Issues Addressed:

- Closes #12959
- Makes some progress towards #10204

## Tests Performed:

- Ran the query without errors/warnings.

## How to Test the Changes:

1. Run the query.
2. Start the server if you want to be thorough.
3. Verify that the existing entries' values have not been changed.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
